index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title> 
+    <link rel="stylesheet" href="style.css">
+
+    <!-- Link to FontAwesome page -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" 
+    integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+    crossorigin="anonymous"/>
+
+    <!-- Font link to font family: Playfair Display -->
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <!-- Font link to font family: Poppins -->
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    
+    <!-- Hamburguer button -->
+    <div class="option-menu">
+        <i class="fas fa-bars"></i>
+    </div>
```

Comment:
- [ ] [REQUIRED] - Please add a nav tag to store the hamburger button for semantic reasons. For example:
![image](https://user-images.githubusercontent.com/61767582/176800104-545dc51d-248e-4a06-a864-bddcbc050231.png)
This will increase the accessibility for the screen readers and you'll follow the best practices

- [ ] [OPTIONAL] - I suggest using a `button` tag instead of a `div` for the hamburger button because it will increase the accessibility for screen readers :slightly_smiling_face:

Reply:
**Ok, thanks for your comments.**
In a moment I will send to you my correction.
**Are you available for a zoom call?** I would like to review the next point about screen dimensions.   @leopiresgoss